### PR TITLE
cat-786 lower the viper threshold from 5 to 2

### PR DESCRIPTION
**Pull request by kkonuganti**

### CODE REVIEW

File: src/test/resources/application-test.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 spring:
-  profiles:
-    include: stdout
-
   security:
     oauth2:
       resourceserver:
         jwt:
           public-key-location: classpath:local-public-key.pub
+  config:
```
Comment (Mjwillis) [inline src/test/resources/application-test.yml:7]: why this change?

**Review by Mjwillis — APPROVED** [review]
LGTM, suspect @steverendell should take a look - although thinking I might have even written this stuff many moons ago

File: src/test/resources/application-test.yml (hunk 1 above)
Comment (kkonuganti) [reply src/test/resources/application-test.yml:7]: it was suggested by spring as the old config style is deprecated.
